Example/EditorDemo/WPTestLegacyEditorViewController.m

```diff
@@ -41,14 +39,14 @@ - (void)viewDidLoad
 - (void)customizeAppearance
 {
     [super customizeAppearance];
-    [self setTitleFont:[WPFontManager notoBoldFontOfSize:24.0]];
-    [self setTitleColor:[WPStyleGuide darkGrey]];
+//    [self setTitleFont:[WPFontManager notoBoldFontOfSize:24.0]];
+//    [self setTitleColor:[WPStyleGuide darkGrey]];
     [self setBodyFont:[UIFont fontWithName: @"Menlo-Regular" size:14.0f]];
-    [self setBodyColor:[WPStyleGuide darkGrey]];
-    [self setPlaceholderColor:[WPStyleGuide textFieldPlaceholderGrey]];
-    [self setSeparatorColor:[WPStyleGuide greyLighten20]];
-
-    [[WPLegacyEditorFormatToolbar appearance] setTintColor:[WPStyleGuide greyLighten10]];
+//    [self setBodyColor:[WPStyleGuide darkGrey]];
+//    [self setPlaceholderColor:[WPStyleGuide textFieldPlaceholderGrey]];
+//    [self setSeparatorColor:[WPStyleGuide greyLighten20]];
+//
+//    [[WPLegacyEditorFormatToolbar appearance] setTintColor:[WPStyleGuide greyLighten10]];
```

Comment:
Maybe add a descriptive TODO: or other comment here reminding us about the shared pod dependency being removed?